src/main/scala/io/iohk/praos/domain/BlockchainState.scala

```diff
@@ -10,4 +10,11 @@ case class BlockchainState(fullBlockchain: Blockchain, receivedChains: List[Bloc
   lazy val maybeHeadBlockHash: Option[ByteString] = fullBlockchain.lastOption.map(_.blockHash)
 }
 
+object BlockchainState {
+  def receiveChain(state: BlockchainState, newChain: Blockchain): BlockchainState = {
```

Comment:
Should we add a `TODO` comment regarding the pending validation of `newChain`?

Reply:
:+1: